### ISSUE TITLE
jupyter not found

### ISSUE THREAD

**Issue by frgomes** [frgomes/bash-scripts#1]
Apparently, install-jupyter.sh does everything it should do and always did well.
However, command ``jupyter`` is not available in the environment.

After some investigation, it became clear what the problem was:
1. I've removed Debian packages ``virtualenv``, ``virtualenv-wrapper`` and friends;
2. I'm now using the simple, barebones, replacement for commands ``workon`` and ``deactivate``, i.e.: instead of using these commands provided by ``virtualenv-wrapper``, I'm now using some shell functions which are small, very simple and avoid pulling a full blown of undesirable dependencies. In particular, in server and restricted environments, we are definitely not interested on lots of dependencies pulled by ``virtualenv`` and friends.
3. ``pip3 install --verbose jupyter`` revealed that ``jupyter`` is under ``$HOME/.local/bin``

So, in a nutshell, we need to make sure that ``$HOME/.local/bin`` is in $PATH.
